stdlib/public/core/Substring.swift

```diff
@@ -97,9 +97,15 @@ public struct Substring {
   @usableFromInline
   internal var _slice: Slice<String>
 
-  @inlinable @inline(__always)
```

Comment:
Expected?

Reply:
Yup, this is no longer trivial now and we don't want to force the optimizer's hand. The optimizer's probably better off not inlining this call and inlining the `scalarAlign` calls.